spec/hashie/mash_spec.rb

```diff
@@ -676,5 +676,14 @@ class SubMash < Hashie::Mash
     it 'does not overwrite values' do
       expect(subject.reverse_merge(a: 5).a).to eq subject.a
     end
+
+    context 'when using with subclass' do
+      let(:subclass) { Class.new(Hashie::Mash) }
+      subject { subclass.new(a: 1) }
+
+      it 'creates an instance of subclass' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.